release.config.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+    branches: [
+        {
+            name: "master",
+            level: "minor",
+            devDependencies: ["@zowe/imperative", "@zowe/cli"]
+        },
+        // {
+        //     name: "zowe-v1-lts",
+        //     level: "patch",
+        //     devDependencies: ["@zowe/imperative", "@zowe/cli"]
+        // },
+        // {
+        //     name: "next",
+        //     prerelease: true,
+        //     devDependencies: ["@zowe/imperative", "@zowe/cli"]
+        // },
+    ],
+    plugins: [
+        "@octorelease/changelog",
+        ["@octorelease/npm", {
+            aliasTags: {
+                latest: ["next"]
+            },
```

Comment:
The `aliasTags` property should be removed if we want to continue publishing 2 separate tags:
![image](https://user-images.githubusercontent.com/22344007/228039262-cb91b54a-63d5-4d06-8dd6-49d056c52b32.png)

We may also want to uncomment the block for the `next` branch above, to continue publishing a prerelease tag for now.